bigquery_etl/metadata/validate_metadata.py

```diff
@@ -130,6 +192,13 @@ def validate(target):
                     ):
                         failed = True
 
+                    if not validate_shredder_mitigation(
+                        query_file_path=root,
```

Comment:
`query_file_path` is a somewhat misleading name because it's the query dir instead and gets `query.sql` appended to it.  I would recommend renaming it to something like `query_dir`